cmd/devenv/registry/registry.go

```diff
@@ -81,6 +81,21 @@ func (o *Options) parseRegistryPath(conf *box.DevelopmentRegistries, devenvName
 	return buf.String(), nil
 }
 
+func (o *Options) getDevenvType(ctx context.Context, b *box.Config) (kubernetesruntime.RuntimeType, error) {
+	conf, err := config.LoadConfig(ctx)
+	if err != nil {
+		return "", errors.Wrap(err, "failed to load config")
+	}
+	kr, err := devenvutil.EnsureDevenvRunning(ctx, conf, b)
+
+	if err != nil {
+		return "", errors.Wrap(err, "failed to lookup current devenv type")
+	}
```

Comment:
super nitpicky:

```suggestion

	kr, err := devenvutil.EnsureDevenvRunning(ctx, conf, b)
	if err != nil {
		return "", errors.Wrap(err, "failed to lookup current devenv type")
	}
```

I always try to keep err checks right below the assignment